pkg/local_object_storage/blobstor/internal/blobstortest/delete.go

```diff
@@ -0,0 +1,82 @@
+package blobstortest
+
+import (
+	"testing"
+
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor/common"
+	apistatus "github.com/nspcc-dev/neofs-sdk-go/client/status"
+	oidtest "github.com/nspcc-dev/neofs-sdk-go/object/id/test"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDelete(t *testing.T, cons Constructor, min, max uint64) {
+	s := cons(t)
+	require.NoError(t, s.Open(false))
+	require.NoError(t, s.Init())
+	t.Cleanup(func() { require.NoError(t, s.Close()) })
+
+	objects := prepare(t, 4, s, min, max)
+
+	t.Run("delete non-existent", func(t *testing.T) {
```

Comment:
can we run all the "not found" tests before the `prepare` call? just to be sure that object is missing

Reply:
The purpose it to run this on a dirty storage so that we don't just delete a random object.